ccan/list/list.h

```diff
@@ -126,6 +126,7 @@ struct list_node *list_check_node(const struct list_node *n,
  * Example:
  *	static LIST_HEAD(my_global_list);
  */
+#undef LIST_HEAD
```

Comment:
You should not just #undef a macro. The question is why is sys/queue.h included?